src/pages/IdentityVerificationPage/IdentityVerificationSelectPage.tsx

```
@@ -1,146 +1,355 @@
+          {slot.status === "success" && (
+            <CheckCircle className="text-green-500" size={20} />
+          )}
+          {slot.status === "error" && (
+            <XCircle className="text-red-500" size={20} />
+          )}
+
+          {slot.file ? (
+            <button
+              onClick={() => onDelete(index)}
+              className="rounded-full bg-red-100 p-1 text-red-600 hover:bg-red-200"
+            >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)